src/explore-education-statistics-admin/src/components/AdminDashboardPublications.tsx

```diff
@@ -19,7 +19,7 @@ const AdminDashboardPublications = ({
           </dt>
           <dd className="govuk-summary-list__value">
             <Link to="/methodology/{publication.methodology.id}">
-              {publication.methodology.title}
+              {publication.methodology.label}
```

Comment:
Wondering if this should remain conistient with the db schema?

Reply:
I know where you're coming from, but my main driver here is that I have a UI type called IdLabelPair that I will be using everywhere (well, everywhere that we just need an id and a label anyway!  Like select options, checkbox fields, any links with the names of things as their text and the ids of things in their hrefs).

If you'd prefer it, I can have this to be returned as {"id" and "title"} and transform it when it comes back from the API response to {"id" and "label"} in the service itself, although then I'd need 2 structure definitions for the responses... hmmmm

Reply:
This will change as per your suggestion, as the way that the API call works will be much more in line with the original domain objects.